### ISSUE TITLE
Confidence intervals for multilabel scenario

### ISSUE THREAD

**Issue by vera-bernhard** [jacobgil/confidenceinterval#13]
I was wondering if you have any suggestion on how to compute confidence intervals for a multiclass, multilabel scenario, i.e. where one sample can have up to n labels.

E. g. The data looking like the following where the labels are onehot encoded:
y_true = [[1, 0, 1], [0, 1, 1], [1, 1, 0], [0, 0, 1], [0, 1, 1], [1, 1, 1]]
y_pred = [[1,1, 0], [0, 1, 0], [1, 0, 0], [1, 0, 1], [0, 1, 1], [1, 1, 1]]

I can see when measuring the performance per single label, it can be treated like a binary classification problem (having that label or not) like one does in a multiclass, single label scenario.
E.g. for Label 1, one would look at
y_true = [1, 0, 1, 0, 0, 1]
y_pred = [1, 0, 1, 1, 0, 1]

However, how does one though calculate the confidence intervals for the micro or macro F1 over all labels in a multilabel scenario? Any guidance on how to use this package to do that?

**Comment by jacobgil** [comment]
I think bootstrapping would be best for this.
Is there an existing metric you are using for computing the micro or macro F1 for this, without confidence intervals ?

Here's the example from the readme, you can replace `sklearn.metrics.balanced_accuracy_score` with a custom function that accepts (ground_truth_labels, predictions) and returns a number.

```python
from confidenceinterval.bootstrap import bootstrap_ci
# You can specify a random generator for reproducability, or pass None
random_generator = np.random.default_rng()
bootstrap_ci(y_true=y_true,
             y_pred=y_pred,
             metric=sklearn.metrics.balanced_accuracy_score,
             confidence_level=0.95,
             n_resamples=9999,
             method='bootstrap_bca',
             random_state=random_generator)
```

**Comment by vera-bernhard** [comment]
Thank you, that answered my question. I'm actually using the f1 score from sklearn.
